ldi-orchestrator/ldio-connectors/ldio-amqp/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/AmqpInConfigKeys.java

```diff
@@ -0,0 +1,18 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldio.config;
+
+public class AmqpInConfigKeys {
```

Comment:
The idea of this class in kafka was to group the configuration keys but this is actually a constants class as you are mixing config keys with constants from your errors and regexes.